editor/src/components/navigator/navigator-item/navigator-item-components.tsx

```diff
@@ -20,11 +20,17 @@ import type { SelectionLocked } from '../../canvas/canvas-types'
 
 export const NavigatorHintCircleDiameter = 8
 
+const outletAwareBackgroundColor = (
```

Comment:
This should probably just be written as a regular function, I found it a bit confusing written like this.